azsettings/clouds.go

```diff
@@ -6,8 +6,6 @@ const (
 	AzurePublic       = "AzureCloud"
 	AzureChina        = "AzureChinaCloud"
 	AzureUSGovernment = "AzureUSGovernment"
-	AzureGermany      = "AzureGermanCloud"
-	AzureCustomized   = "AzureCustomizedCloud"
```

Comment:
this is a breaking change, `grafana/grafana` is still using these exported variables, for example: https://github.com/grafana/grafana/blob/main/pkg/tsdb/azuremonitor/credentials.go#L55

Reply:
@kostrse may need to update all other places to remove the reference, as Germany cloud was removed last year (no such cloud any more).

Reply:
The Germany cloud has been deprecated in 2020 closed on October 29. 2021 and cannot be used.
https://learn.microsoft.com/en-us/azure/active-directory/develop/authentication-national-cloud

All references to it in the code and UI can be removed in Grafana and ADX with a release note.

Users should not be impacted.

Reply:
Thanks @andresmgot ! 
qq: is Grafana core always picking up the latest SDK version?
@kostrse let's work with @andresmgot to learn on the version control and follow the best practice on release.